src/elements/content-sidebar/activity-feed/common/activity-message/CollapsableMessage.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import ToggleMoreLessButton from './ToggleMoreLessButton';
+import './ActivityMessage.scss';
+
+type Props = {
+    children: React.Node,
+};
+
+// this is for backward support for IE11
+const MAX_MESSAGE_HEIGHT = 140;
+
+export default function CollapsableMessage({ children }: Props) {
+    const [isCollapsed, setIsCollapsed] = React.useState(true);
+    const [shouldCollapse, setShouldCollapse] = React.useState(false);
+    const messageContainer = React.useRef<HTMLDivElement | null>(null);
+
+    React.useLayoutEffect(() => {
+        if (messageContainer.current) {
+            const { clientHeight, scrollHeight } = messageContainer.current;
+            setShouldCollapse(clientHeight !== scrollHeight || clientHeight > MAX_MESSAGE_HEIGHT);
+        }
+    }, []);
+
+    return (
+        <>
+            <div
+                className={classNames({
+                    'bcs-ActivityMessage-collapsed': isCollapsed,
+                })}
+                style={{ maxHeight: isCollapsed ? MAX_MESSAGE_HEIGHT : 'none' }}
```

Comment:
Do we need to use inline styling if we use `isCollapsed` in the classname?

Reply:
Actually, we don't need it anymore. It was for backward compatibility with IE11. But as @jstoffan pointed out it's not required.